packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.18+1
+
```

Comment:
This needs to follow the style guide linked from the checklist.

Reply:
Sorry, this ended up on the wrong line somehow. The comment was about the changelog message, not the version (which was correct before).